project-base/app/src/DataFixtures/Demo/ComplaintDataFixture.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\DataFixtures\Demo;
+
+use App\Model\Customer\User\CustomerUser;
+use App\Model\Order\Item\OrderItem;
+use App\Model\Order\Order;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Persistence\ObjectManager;
+use Shopsys\FrameworkBundle\Component\CustomerUploadedFile\CustomerUploadedFileDataFactory;
+use Shopsys\FrameworkBundle\Component\DataFixture\AbstractReferenceFixture;
+use Shopsys\FrameworkBundle\Model\Complaint\Complaint;
+use Shopsys\FrameworkBundle\Model\Complaint\ComplaintDataFactory;
+use Shopsys\FrameworkBundle\Model\Complaint\ComplaintItemData;
+use Shopsys\FrameworkBundle\Model\Complaint\ComplaintItemDataFactory;
+use Shopsys\FrameworkBundle\Model\Complaint\ComplaintNumberSequenceRepository;
+use Shopsys\FrontendApiBundle\Model\Complaint\ComplaintApiFacade;
+
+class ComplaintDataFixture extends AbstractReferenceFixture implements DependentFixtureInterface
```

Comment:
it is a common practice that we set entity uuids in the data fixtures to ease (both automated and manual) testing, see e.g. https://github.com/shopsys/shopsys/blob/15.0/project-base/app/src/DataFixtures/Demo/ArticleDataFixture.php#L31